lib/jlc-parts-engine.ts

```diff
@@ -195,6 +194,16 @@ export const jlcPartsEngine: PartsEngine = {
       return {
         jlcpcb: leds.map((l: any) => `C${l.lcsc}`).slice(0, 3),
       }
+    } else if (
+      sourceComponent.type === "source_component" &&
+      sourceComponent.ftype === "simple_fuse"
+    ) {
+      const { fuses } = await getJlcPartsCached("fuses", {
+        package: footprinterString,
```

Comment:
can you use any additional data from the fuse? you're only filtering based on footprint